parser/src/README.md

```diff
@@ -0,0 +1,34 @@
+# RustPython/parser/src
```

Comment:
Maybe this could go in a section in `parser/README.md`? Similar to how we have the project directory structure in the root `README.md`.

Reply:
I wasn't sure which one made sense. I'll fix and update the PR in a bit.